From: Jan Engelhardt <jengelh@inai.de>
Date: 2020-06-11 15:48:28.422839234 +0200

gzdoom is the only user of its own libSPIRV.so, so there is no need for a
shared library.

---
 libraries/glslang/spirv/CMakeLists.txt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gzdoom-g4.4.0/libraries/glslang/spirv/CMakeLists.txt
===================================================================
--- gzdoom-g4.4.0.orig/libraries/glslang/spirv/CMakeLists.txt
+++ gzdoom-g4.4.0/libraries/glslang/spirv/CMakeLists.txt
@@ -49,7 +49,7 @@ set(SPVREMAP_HEADERS
     SPVRemapper.h
     doc.h)
 
-add_library(SPIRV ${LIB_TYPE} ${SOURCES} ${HEADERS})
+add_library(SPIRV STATIC ${LIB_TYPE} ${SOURCES} ${HEADERS})
 set_property(TARGET SPIRV PROPERTY FOLDER glslang)
 set_property(TARGET SPIRV PROPERTY POSITION_INDEPENDENT_CODE ON)
 target_include_directories(SPIRV PUBLIC
